torch_frame/data/dataset.py

```diff
@@ -260,6 +281,13 @@ class Dataset(ABC):
             values are tensors such as tokens.
             :obj:`batch_size` specifies the mini-batch size for
             :obj:`text_tokenizer`. (default: :obj:`None`)
+        col_to_time_format (Union[str, Dict[str, str]], optional): A
+            dictionary or a string specifying the format for the timestamp
+            columns. If a string is specified, then the same separator will
+            be used throughout all the multi-categorical columns. If a
+            dictionary is given, we use a separator specified for each
+            column. If not specified, pandas's internal to_datetime function
+            will be used to auto parse time columns. (default: None)
```

Comment:
It'd be great if you can provide a link on how to write `time_format`. Also, the doc-string seems to contain multi-categorical information. Can you re-write?